pulp_ansible/app/galaxy/serializers.py

```diff
@@ -80,6 +80,33 @@ def get_versions_url(self, obj):
             )
         )
 
+    def get_href(self, obj):
+        """
+        Get href.
+        """
+        return "{hostname}/pulp_ansible/galaxy/{path}/api/v2/collections/{namespace}/{name}/".format(
+            path=obj.path,
+            hostname=settings.ANSIBLE_API_HOSTNAME,
+            namespace=obj.namespace,
+            name=obj.name,
+        )
+
+    class Meta:
+        fields = ("name", "namespace", "href", "versions_url")
+        model = CollectionVersion
```

Comment:
This looks right, but here you can see how it motivates the name `GalaxyCollectionVersionSerializer`. I'm not sure what would be best. I'm ok w/ the change as you've written it also.

Reply:
It should be `Collection`. I've updated it.